mantidproject/mantid
# Muon Sample log copying

Ensure that the sample logs are copied from the MuonPhaseQuad input to the output.

**To test:**

* Load some data into muon analysis (MUSR 68389)
* Open the frequency domain anlysis GUI
* Change the workspace to phaseQuad
* Press the calculate button
* In the grouped workspace a PhaseQuad;FTT output has been added.
* Ensure this workspace has the same sample logs as the `MuonAnalysis` workspace.

Fixes #21885 

**Release Notes** 
*Does not need to be in the release notes.*


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
